### PR TITLE
Database

### PR DESCRIPTION
**Pull request by aberracja**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,8 +9,8 @@ gem 'print_clerk' , path: "../print_clerk"
 
 gem 'post_clerk' ,  github: "rubyclerks/post_clerk"
 
-gem "office_clerk", github: "rubyclerks/office_clerk"
-#gem "office_clerk", path: "../office_clerk"
+#gem "office_clerk", github: "rubyclerks/office_clerk"
+gem "office_clerk", path: "../office_clerk"
```
Comment (pjar) [inline Gemfile]: Use remote gem

**Comment by Czarrrna** [comment]
Can we merge?

cc/ @dancinglightning @pjar @lsolniczek

**Comment by rubydesign** [comment]
the gemfile still points to a :path .  and you should make a pr to my repo so i can accept it

**Comment by Czarrrna** [comment]
Ok, now it's fixed.

**Comment by rubydesign** [comment]
ok then you can make the pr to rubyclerks/sales_clerk

**Comment by Czarrrna** [comment]
We did.

**Pull request closed without merging**